src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,31 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
+Animal.alive = [];
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health = 100, hidden = false) {
+    super();
+    this.hidden = hidden;
+  }
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(object) {
+    if (object.hidden === false) {
```

Comment:
what about if animal is `Carnivore`?
you should bite only `Herbivore`
<img width="828" alt="image" src="https://github.com/mate-academy/js_herbivores_and_carnivores/assets/6568745/f1b2c52a-742c-46f3-8279-0c897bedf3c6">